src/runtime_src/core/include/xgq_cmd_vmr.h

```diff
@@ -47,12 +47,14 @@
 #define XGQ_CLOCK_WIZ_MAX_RES           4
 
 enum xgq_cmd_sensor_page_id {
-	XGQ_CMD_SENSOR_PID_ALL		= 0x0,
+	XGQ_CMD_SENSOR_PID_GET_SIZE	= 0x0,
```

Comment:
Please update XGQ1.0 spec as well.
https://confluence.xilinx.com/pages/viewpage.action?pageId=302345261#XGQ1.0CommandandQueueSpecification(WIP)-GetLogPage

Reply:
ok

Reply:
I've updated the spec with new page IDs.
Once this is merged, we will also need to sync-up the VMR header files and code.

I think this could be optimized in the future by passing real opcode, but let's stick with current solution for now.

Reply:
given we don't want to add the get_size, should we remove this?